tests/unit/testHarnessTest.js

```diff
@@ -0,0 +1,50 @@
+define([
+    '$',
+    'text!fixtures/pinny.html',
+    'iframeTestHarness'
+], function(_$, fixture, iframeTestHarness) {
```

Comment:
Again, we were using this before. We moved that code out. Can probably remove it.